<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SecondaryWindow</class>
 <widget class="QMainWindow" name="SecondaryWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>497</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QRadioButton" name="Scheme_A_Selection">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>240</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Scheme A</string>
    </property>
   </widget>
   <widget class="QScrollBar" name="Final_scroll">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>160</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="Midterm1_spin">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>100</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="Final_spin">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>180</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="Course_Selection">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>271</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>PIC 10C: Advanced Programming</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>PIC 10B: Intermediate Programming</string>
     </property>
    </item>
   </widget>
   <widget class="QRadioButton" name="Scheme_B_Selection">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>260</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Scheme B</string>
    </property>
   </widget>
   <widget class="QLabel" name="Final_Score_Display">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>250</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {background-color: white}</string>
    </property>
    <property name="text">
     <string>         0</string>
    </property>
   </widget>
   <widget class="QScrollBar" name="Midterm1_scroll">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>80</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="Midterm1_label">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>60</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Midterm</string>
    </property>
   </widget>
   <widget class="QLabel" name="Final_label">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>140</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Final Exam</string>
    </property>
   </widget>
   <widget class="QLabel" name="Course_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Course:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Assignment2_label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Grade Calculator</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="Assignment2_spin">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Assignment1_label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Siete y Medio</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="Assignment1_spin">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QScrollBar" name="Assignment1_scroll">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QScrollBar" name="Assignment2_scroll">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="Assignment3_label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>A Ring Queue</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="Assignment3_spin">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QScrollBar" name="Assignment3_scroll">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QScrollBar" name="Project_scroll">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="Project_label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>240</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Project    </string>
    </property>
   </widget>
   <widget class="QSpinBox" name="Project_spin">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
